internal/proto/buf.yaml

```diff
@@ -1,4 +1,8 @@
 version: v1
+name: buf.build/hashicorp/boundary
```

Comment:
This module does not exist on buf.build, but I have reserved the `hashicorp` organization just in case: https://buf.build/hashicorp. There's no need to create this to use the BSR anonymously, but it could be useful if we wanted to share protobuf definitions at some point.